### DISCUSSION TITLE
Cannot edit load order in 1.4.0?

### DISCUSSION THREAD

**Discussion by kenanthebarbarian** [X2CommunityCore/xcom2-launcher#253] [General]
Upgraded to 1.4.0, and there no longer appears to be a column for load order. Am I missing something?

**Comment by RevZero** [comment]
The order column is now hidden by default, because it does not represent the "real" loader order and relying on it is not recommended. Check [Working with the mod list](https://github.com/X2CommunityCore/xcom2-launcher/wiki/Basic-usage#modlist) about how to hide/show columns and some other fancy stuff if you want to use it anyway.

Regarding load order:   
There are other factors like subscription date that (seem to) influence how XCOM loads mods. There were [some discussions](https://github.com/X2CommunityCore/xcom2-launcher/issues/173#issuecomment-570076752) about this, but nothing really conclusive AFAIK.

**Comment by kenanthebarbarian** [comment]
Thanks for the explanation!
